### ISSUE TITLE
There is an issue for Rails 4.2.5.2

### ISSUE THREAD

**Issue by rpmilevskiy** [ilyakatz/data-migrate#29]
Maybe it's not related to rails version. But it seems like that.
I run 'rake db:migrate:with_data'

NoMethodError: undefined method `group_by' for "db/data":String
/home/ruslan/.rvm/gems/ruby-2.3.0/gems/activerecord-4.2.5.2/lib/active_record/migration.rb:1016:in `validate'
/home/ruslan/.rvm/gems/ruby-2.3.0/gems/activerecord-4.2.5.2/lib/active_record/migration.rb:923:in `initialize'
/home/ruslan/.rvm/gems/ruby-2.3.0/gems/data_migrate-2.0.1/tasks/databases.rake:315:in `new'
/home/ruslan/.rvm/gems/ruby-2.3.0/gems/data_migrate-2.0.1/tasks/databases.rake:315:in `pending_data_migrations'
/home/ruslan/.rvm/gems/ruby-2.3.0/gems/data_migrate-2.0.1/tasks/databases.rake:311:in `pending_migrations'
/home/ruslan/.rvm/gems/ruby-2.3.0/gems/data_migrate-2.0.1/tasks/databases.rake:12:in `block (3 levels) in <top (required)>'
/home/ruslan/.rvm/gems/ruby-2.3.0/gems/airbrake-5.1.0/lib/airbrake/rake/task_ext.rb:19:in `execute'
/home/ruslan/.rvm/gems/ruby-2.3.0/bin/ruby_executable_hooks:15:in `eval'
/home/ruslan/.rvm/gems/ruby-2.3.0/bin/ruby_executable_hooks:15:in `<main>'

**Comment by ilyakatz** [comment]
I actually, have a PR https://github.com/ilyakatz/data-migrate/pull/25 that will fix this issue. Could you try it out and let me know if it fixes the situation for you

```ruby
gem "data_migrate", github: "execonline-inc/data-migrate",
  branch: "multiple_db_migrations"
```

**Comment by ilyakatz** [comment]
@rpmilevskiy I published new version 2.1.0 that should take care of this problem. Can you give it a try please

**Comment by rpmilevskiy** [comment]
@ilyakatz Yes, it is fixed now. Thanks!

But I have found another two problems. Please see other issue https://github.com/ilyakatz/data-migrate/issues/31
